cli/build/build_approve.go

```diff
@@ -19,6 +19,7 @@ var buildApproveCmd = &cli.Command{
 }
 
 func buildApprove(c *cli.Context) (err error) {
+	common.SetupConsoleLogger(c)
```

Comment:
Isn't there something like a global hook / init fnc for `github.com/urfave/cli/v2` to call this logger setup?

Reply:
from checking the docs there only seems to be an option for commands called `Before`:

```go
       Before: func(c *cli.Context) error {
          fmt.Fprintf(c.App.Writer, "brace for impact\n")
          return nil
        },
```

But I doesn't look like we could save code that way.

Reply:
Ohh, from the api it seems as we could add it to the app as global before hook as well.

Reply:
I'll have a look - that's why it's not merged jet :D

Reply:
Yeah no pressure. Just wanted to inform you about my investigations on it. 😉

Reply:
Before did the job ...